src/rdb.h

```diff
@@ -35,10 +35,11 @@
 
 /* TBD: include only necessary headers. */
 #include "server.h"
+#include "cluster.h"
```

Comment:
This shouldn't be included here, since there is no dependencies in rdb.h for cluster stuff. Probably needs to be in some other file, probably rdb.c

Reply:
Removed.